OasisDEX/oasis-borrow
# DAI allowance

# [DAI allowance](https://app.clubhouse.io/oazo-apps/story/2382/other-actions-second-iteration-for-other-actions-add-dai-allowance-steps)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
- DAI allowance steps added 
  
## How to test 🧪
- Navigate to multiply valut 
- Go to other actions
- Try depositing DAI
- On progress you should be taken to `Dai Allowance` step
    
## Definition of done ✔️
